cmd/ocm-backplane/cloud/common.go

```diff
@@ -24,14 +24,18 @@ import (
 	logger "github.com/sirupsen/logrus"
 )
 
-const OldFlowSupportRole = "role/RH-Technical-Support-Access"
+const (
+	OldFlowSupportRole         = "role/RH-Technical-Support-Access"
+	CustomerRoleArnName        = "Target-Role-Arn"
+	CustomerArnRoleSessionName = "RH-SRE"
```

Comment:
This flow is already defined in `bp-api` 
https://gitlab.cee.redhat.com/service/backplane-api/-/blob/master/configs/cloud-config.yml?ref_type=heads#L9

Reply:
Moving some of our slack conversation here for others who may look:

The assume role calls for the isolated flow occur here in the CLI. This is because SREs are using their local credentials which we don't want to send over the wire. There's a separate PR for backplane-api with the old flow

Reply:
@AlexVulaj backplane today has a UUID used for service account creation on clusters, this UUID is mapped to an SRE. Are we able to use the same UUID?

Reply:
@jharrington22  I'd be in favor of this, though I'm unfamiliar with the UUID you speak of. Do you know where this UUID is generated or how we do that mapping?

Reply:
Never mind - found it in backplane API. I'm pushing a change now.